.travis.yml

```diff
@@ -1,12 +1,13 @@
 language: rust
 sudo: false
-rust:
-  - nightly
-env:
-  - FEATURES=""
-  - FEATURES="unstable"
+matrix:
+  include:
+    - rust: nightly
+    - rust: nightly
+      env:
+      - FEATURES="unstable"
```

Comment:
This is embarrassing! However, I'd like to implement it like I did [for sxd-xpath](https://github.com/shepmaster/sxd-xpath/blob/237e01e1e2b44d8440a9fce85350b22ec6ed28f8/.travis.yml#L3-L18). That is, let Travis build out the matrix, as much as possible.

Feel free to just drop the travis changes if you want me to do it; now that you've pointed out that I wasn't actually testing what I thought, I can fix it :blush: